tests/integration/test_cli.py

```diff
@@ -43,17 +46,26 @@
 @pytest.mark.parametrize(
     "bad_url,file_id,output_dir,max_wait_time,expected_exception",
     [
-        (True, "1", EXAMPLE_FOLDER, "60", ApiNotReachable),
-        (False, "1", EXAMPLE_FOLDER, "60", None),
-        (False, "2", EXAMPLE_FOLDER, "60", BadResponseCodeError),
-        (False, "10s", EXAMPLE_FOLDER, "60", MaxWaitTimeExceeded),
-        (False, "1", "/this_path/", "60", DirectoryNotExist),
+        (True, "downloadable", EXAMPLE_FOLDER, "60", ApiNotReachable),
+        (False, "downloadable", EXAMPLE_FOLDER, "60", None),
+        (False, "not_downloadable", EXAMPLE_FOLDER, "60", BadResponseCodeError),
+        (False, "retry", EXAMPLE_FOLDER, "60", MaxWaitTimeExceeded),
+        (False, "downloadable", "/this_path/", "60", DirectoryNotExist),
     ],
 )
-def test_download(bad_url, file_id, output_dir, max_wait_time, expected_exception):
-
+def test_download(
+    bad_url,
+    file_id,
+    output_dir,
+    max_wait_time,
+    expected_exception,
+    s3_fixture,  # noqa F811
+):
     """Test the download of a file, expects Abort, if the file was not found"""
-    with MockAPIContainer() as api:
+
+    with MockAPIContainer(
+        s3_download_url=get_presigned_download_url(s3_config=s3_fixture.config)
+    ) as api:
```

Comment:
You can directly use the storage dao from the s3_fixture:

```suggestion
    downloadable_file = state.FILES["file_in_outbox"]
    download_url = s3_fixture.storage.get_object_download_url(
        bucket_id=downloadable_file.grouping_label,
        object_id=downloadable_file.file_id,
        expires_after=60,
    )

    with MockAPIContainer(s3_download_url=download_url) as api:
```

Reply:
Same for the upload.